embedded-graphics/src/style.rs

```diff
@@ -6,8 +6,6 @@ use pixelcolor::PixelColor;
 #[derive(Debug, Copy, Clone)]
 pub struct Style<P: PixelColor> {
     /// Fill colour of the object
-    ///
-    /// Has no effect for fonts
```

Comment:
I think this doc should mention that it sets the _background_ colour for a font. There's some semantic difference between text background and fill colour, so it's worth mentioning.